python/libsettings.pyx

```diff
@@ -150,15 +150,15 @@ cdef class Settings:
 
     def write(self, section, name, value):
         return settings_write_str(self.ctx,
-                                  bytes(section),
-                                  bytes(name),
-                                  bytes(str(value)))
+                                  bytearray(section, 'utf8'),
```

Comment:
When I did the python3 update for the command line settings application, I asked somebody (probably @silverjam ) what the settings key/value encoding should be, and he said ascii. Back then I put it into 2 constants (one for keys and one for values) in the cli application. Maybe getting the encoding (whatever it is) should be part of the public API of libsettings so it can be easily changed if needed and user applications (console and in the future also settings.py) can get it? 

Also coordinate this with console's settings PR (at the moment there is one encoding/decoding command that uses hard-coded ascii).

Reply:
`ascii` it is then.

Reply:
Need to think about the  encoding from API. I am not going to mix it in here anymore though.

Reply:
Actually, all these are *inputs* for the `libsettings` so the client needs to provide the encoding of the argument string. I need to add the argument for that.